README.md

```diff
@@ -215,8 +215,6 @@ Any API call returns a response object which is accessible by dot notation
 and `response['foo']['bar']`. Expected keys for all types of responses are defined,
 and any attempt to access an unknown key will cause `NoMethodError` exception.
 
-These strict model checks can be disabled by setting `Plaid.relaxed_models = true`.
-
```

Comment:
My thought is we should just delete the references to it in the Readme as the flag is still useful for running tests.